src/App.jsx

```diff
@@ -268,10 +268,7 @@ class App extends Component {
               resourceBaseHref: 'https://wevote.us/',
             });
           }
-          const voterWeVoteId = VoterStore.getVoterWeVoteId();
-          tracker.start({
-            userID: voterWeVoteId,
-          });
+          tracker.start();
```

Comment:
Hi @Nyette, I don't see anywhere else in the code where the userId is set. Are you sure we want to remove this?

Reply:
`userID` is being set twice, unless I misunderstood. See [this line](https://github.com/wevote/WebApp/blob/develop/src/js/stores/VoterStore.js#L1158).